front/app/modules/commercial/native_surveys/admin/components/SurveyBuilderToolbox/ToolboxItem.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// Components
+import { Box, Icon, IconNames, Text } from '@citizenlab/cl2-component-library';
+import { colors } from 'utils/styleUtils';
+
+interface Props {
+  label: string;
+  icon: IconNames;
+}
+
+const StyledBox = styled(Box)`
+  &:hover {
+    background-color: ${colors.emailBg};
+    transition: background-color 80ms ease-out 0s;
+  }
+`;
+
+const ToolboxItem = ({ icon, label }: Props) => {
+  return (
+    <StyledBox
+      width="100%"
+      display="flex"
+      paddingLeft="10px"
+      alignItems="center"
+    >
+      <Icon
+        marginRight="16px"
+        width="20px"
+        height="20px"
+        fill={colors.adminTextColor}
+        name={icon}
+      />
+      <Text color="text" style={{ lineHeight: '1' }}>
+        {label}
```

Comment:
I think the Text component lineHeight is already set to some design standard - do you think we could apply padding or something instead to get the desired effect?